src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,53 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
-);
+export const MovieCard: React.FC<Movie> = ({
+  title,
+  description,
+  imgUrl,
+  imdbUrl,
+}) => {
+  return (
```

Comment:
```suggestion
}) => (
```
you can use shorthand return here